Dockerfile

```diff
@@ -29,13 +29,18 @@ RUN apt-get update \
     sane-utils \
     tesseract-ocr \
     sane-airscan \
+    libsane-hpaio \
   && sed -i \
     's/policy domain="coder" rights="none" pattern="PDF"/policy domain="coder" rights="read | write" pattern="PDF"'/ \
     /etc/ImageMagick-6/policy.xml \
   && sed -i \
     's/policy domain="resource" name="disk" value="1GiB"/policy domain="resource" name="disk" value="8GiB"'/ \
     /etc/ImageMagick-6/policy.xml \
-  && npm install -g npm@8.3.0
+  && npm install -g npm@8.3.0 \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
+
+RUN echo hpaio >> /etc/sane.d/dll.conf
```

Comment:
How about, instead of here, adding something like this to the end...


```Dockerfile
# hplib image
#
# This image adds the HP scanner libs to the image.
# ==============================================================================
FROM scanservjs-core AS scanservjs-hplib
RUN apt-get install -yq \
    libsane-hpaio \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/* \
  && echo hpaio >> /etc/sane.d/dll.conf
```

It won't get included by default but allows users to build their own by specifying `target`. There could be a similar approach for brother images. I haven't tested this but assume it would work.